TicketSwap/solar
# Add icons to Solar

# Description
This will add the icons from Comets into solar

## Changes
- [x] Added all icons
- [x] Changed build process to add `/icons` bundle 
- [x] Added iconography to guidelines in storybook

## How to test
1. run `yarn storybook` to see the new guidelines
2. run `yarn dev-pack`, unzip the package zipfile from `./dist` and place it's content into your own library to see the changes. (Replace the solar package).

## Screenshots
![afbeelding](https://user-images.githubusercontent.com/22071649/142186890-6390ba0f-4302-4929-afb8-f0552ac6a25b.png)
